reana_server/cli.py

```diff
@@ -11,15 +11,13 @@
 import logging
 
 import click
-from reana_commons.config import REANA_LOG_FORMAT, REANA_LOG_LEVEL
 
 from reana_server.scheduler import WorkflowExecutionScheduler
 
 
 @click.command("start-scheduler")
 def start_scheduler():
     """Start a workflow execution scheduler process."""
-    logging.basicConfig(level=REANA_LOG_LEVEL, format=REANA_LOG_FORMAT)
```

Comment:
I think adding `force=True` should fix it. [python 3.8 logging docs](https://docs.python.org/3.8/library/logging.html#logging.basicConfig)

Example:

`logging.basicConfig(level=REANA_LOG_LEVEL, format=REANA_LOG_FORMAT, force=True)`

Reply:
Good catch! Works as expected with less changes